UIKitDZ/UIKitDZ/ViewControllers/SignInViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Анастасия Козлова on 21.09.2022.
+//
+
+import UIKit
+/// Экран входа
+final class SignInViewController: UIViewController, UITextFieldDelegate {
```

Comment:
```suggestion
final class SignInViewController: UIViewController {
```
UITextFieldDelegate в extension